### DISCUSSION TITLE
챕터 2. 웹 최적화

### DISCUSSION THREAD

**Discussion by yujo11** [Frontend-Divers/performance-optimization#10] [함께 얘기하고 싶은 내용]
챕터 2. 웹 최적화

**Comment by Tanney-102** [comment]
DNS에 대해 더 공부해보고 싶네요!
혹시 학습하신 분들 계시면 공유해주시면 좋을 것 같아요!

**Comment by Tanney-102** [comment]
HTTP 파이프라이닝과 멀티플렉싱 차이

**Comment by ritajeong** [comment]
이야기한 내용들
- WEB1,2,3의 차이
 정적 데이터 / 유저들의 참여, 쌍방향 소통 / 분산형, 블록체인 등
- HTTP1.0 / 1.1 / 2.0 의 차이
 3-way handshake / 파이프라이닝 / 멀티플렉싱
- DNS
 계층 시스템, DNS 네임 서버
 127.0.0.1과 localhost:3000 같은 결과인 이유
